docs: update Brook Jeynes profile (#1322)
SSWConsulting/SSW.People.Profiles: Brook-Jeynes/Brook-Jeynes.md

[[imgBadge]]
| ![Zig](../badges/Developer-zig.png)

> Brook Jeynes is a Software Engineer currently working in the Brisbane office.  

Reviewer: **suggestion**: No ending space.

```suggestion
Brook Jeynes is a Software Engineer currently working in the Brisbane office.
```

Author: Intentional line breaks

At SSW, he is predominantly working on [Tina - the world's best headless CMS powered by Markdown and GitHub](https://tina.io).  

> Specifically, the backend side of Tina, TinaCloud, where he writes Golang, React, NodeJS, and interactes with AWS based systems.

Reviewer: suggestion: Smoother wording:

```suggestion
He focuses on the backend side of Tina, TinaCloud, where he writes Golang, React and NodeJS code that interacts with AWS services.
```
